docs/current_docs/manuals/developer/snippets/documentation/python/object.py

```diff
@@ -0,0 +1,11 @@
+from typing import Annotated
+
+from dagger import Doc, object_type
+
+
+@object_type
+class MyModule:
+    """The object represents a single user of the system."""
+
+    name: Annotated[str, Doc("The name of the user.")]
+    age: Annotated[str, Doc("The age of the user.")]
```

Comment:
This creates a constructor automatically, which explains why the constructor's arguments show, but they don't also show up as functions because you need to expose them with `field`:

```suggestion
    name: Annotated[str, Doc("The name of the user.")] = field()
    age: Annotated[str, Doc("The age of the user.")] = field()
```